rdwatch/scoring/views/vector_tile.py

```diff
@@ -526,7 +526,7 @@ def vector_tile(
         evaluation_run_uuid, z, x, y, latest_timestamp
     )
 
-    tile = cache.get(cache_key)
+    tile = None
```

Comment:
```suggestion
    tile = cache.get(cache_key)
```

Leftover from development?